MapboxCoreNavigation/Router.swift

```diff
@@ -50,6 +50,11 @@ public protocol RouterDataSource {
      */
     @objc var location: CLLocation? { get }
     
+    /**
+     If true, the `RouteController` attempts to calculate a more optimal route for the user on an interval defined by `RouteControllerProactiveReroutingInterval`.
+     */
+    @objc var reroutesProactively: Bool { get }
```

Comment:
We’re getting close to being able to refresh the existing route on a timer as well. Should we control both behaviors through a single, more generically named property, on an interval defined by a single, more generically named constant? Or should we keep the two complementary behaviors separate?

Reply:
I think it's OK to keep a separate flag for "reroutesProactively" (although the naming leaves a little to be desired). Faster route polling should happen _directly after_ a directions refresh response, so it's possible we'll be reworking this code again soon.

I don't think we should combine them into one option, to directly answer your question.

Reply:
what does 'refresh the existing route' mean? separate i reckon, by the sound of it.